.gitignore

```diff
@@ -15,10 +15,12 @@ fort.*
 *.log
 
 # Ignore the contents of these directories
-exec/
-bin/
-build/
-install/
+bin*/
+build*/
+exec*/
+install*/
+versions/build.ver
```

Comment:
Why are /versions/*.ver to be be ignored?

Reply:
I have created machine-specific `.ver` files. At build time the specific `.ver` files for the machine get copied to `build.ver` and `run.ver` (to satisfy the NCO requirement). Since these are copies of existing files and not something we want to commit back, we should ignore them in the git status.